bangazonapi/views/product.py

```diff
@@ -268,7 +259,7 @@ def list(self, request):
 
         if number_sold is not None:
             def sold_filter(product):
-                if product.number_sold <= int(number_sold):
+                if product.number_sold >= int(number_sold):
```

Comment:
This is a fix for #21 since it's unrelated to this ticket undo this change and make another pr for that ticket